src/common/store/storage/persistence.slice.ts

```diff
@@ -37,7 +38,7 @@ export const getPersistenceState = createSelector(
 );
 
 export const getPersistentKeys = createSelector(
-  (state: Record<string, PersistState>) => state,
+  (state: Record<string, Optional<PersistState>>) => state,
```

Comment:
What's the reason for this? If necessary, can we use `Partial` (default in TS) instead of `Optional`?

Reply:
Without this the typing of the state breaks for some selectors. Partial doesn't work for some reason. You can probably reproduce locally.